.circleci/config.yml

```diff
@@ -26,7 +26,28 @@ jobs:
                        -scheme 'Automattic-Tracks-OSX' \
                        -destination 'platform=OS X,arch=x86_64' \
                        test  | tee logs/build-and-test.log | xcpretty -r junit
+  CheckVersionConsistency:
+    docker:
+      - image: cimg/base:2020.01
```

Comment:
Using the most basic of [CircleCI's NextGen conveninence images](https://circleci.com/docs/2.0/circleci-images/#next-generation-convenience-images) with just the bare minimum here because we only need to run a very simple shell script and this one is likely one of the fastest to allocate and boot.